templates/zerver/navbar.html

```diff
@@ -12,6 +12,7 @@
     <div class="column-middle-inner">
           <div id="streamlist-toggle" {%if embedded %} style="visibility: hidden"{% endif %}>
             <a href="#" id="streamlist-toggle-button" role="button"><i class="icon-vector-reorder"></i>
+              <span class="ariahide">Streamlist toggle button</span>
```

Comment:
That's a good point, I had initially gone with this approach (https://stackoverflow.com/questions/32631180/link-contain-no-text-showing-error-in-wave-accessibility-evaluation-tool), as it prevents the anchor from being empty. it does seem from https://www.deque.com/blog/text-links-practices-screen-readers/ that aria-label gains precedence on any attribute so it should be the one we use, but I'm not sure if the empty link text causes a different problem, but it surely triggers some accessibility tools (https://purplehippodesign.co.uk/make-font-awesome-accessible/). @jmbowman do you have any thoughts on this?